packages/region-config-resolver/src/regionConfig/resolveRegionConfig.ts

```diff
@@ -0,0 +1,65 @@
+import { Provider } from "@smithy/types";
+
+import { getRealRegion } from "./getRealRegion";
+import { isFipsRegion } from "./isFipsRegion";
+
+/**
+ * @public
+ */
+export interface RegionInputConfig {
+  /**
+   * The AWS region to which this client will send requests
+   */
+  region?: string | Provider<string>;
+
+  /**
+   * Enables FIPS compatible endpoints.
+   */
+  useFipsEndpoint?: boolean | Provider<boolean>;
+}
+
+interface PreviouslyResolved {}
```

Comment:
annotate with internal

Reply:
Why is this interface needed if it's empty?